Composer/packages/client/src/recoilModel/dispatchers/provision.ts

```diff
@@ -135,7 +135,7 @@ export const provisionDispatcher = () => {
           });
 
           // update publishTargets
-          callbackHelpers.set(settingsState(projectId), (settings) => {
+          await callbackHelpers.set(settingsState(projectId), (settings) => {

```

Comment:
recoil set does not need an await. it does things synchronously.

Reply:
remove it